pom.xml

```diff
@@ -76,11 +76,11 @@
         <!-- These are compiled and pass CQs and should be compatible with recommended versions -->
         <!-- to compile against Accumulo 1.8.0, use the accumulo-1.8 profile -->
         <accumulo.version>1.7.2</accumulo.version>
-        <zookeeper.version>3.4.5</zookeeper.version>
+        <zookeeper.version>3.4.6</zookeeper.version>
         <thrift.version>0.9.1</thrift.version>
-        <hadoop.version>2.5.2</hadoop.version>
-        <spark.version>2.0.0</spark.version>
-        <hbase.version>1.2.3</hbase.version>
+        <hadoop.version>2.7.3</hadoop.version>
+        <spark.version>2.1.0</spark.version>
+        <hbase.version>1.1.2</hbase.version>
```

Comment:
yeah, we like to have recent versions, but there's a cost and delay to updating due to eclipse licensing. For most things (e.g. hadoop), the actual runtime version doesn't really matter, as it's a `provided` dependency. For hbase it's slightly trickier since we sometimes bundle it into uber-jars.